Sources/AppFeature/AppDelegate.swift

```diff
@@ -25,92 +25,94 @@ public struct AppDelegateReducer: Reducer {
 
   public init() {}
 
-  public func reduce(into state: inout State, action: Action) -> Effect<Action> {
-    switch action {
-    case .didFinishLaunching:
-      return .run { send in
-        await withThrowingTaskGroup(of: Void.self) { group in
-          group.addTask {
-            for await event in self.userNotifications.delegate() {
-              await send(.userNotifications(event))
+  public var body: some ReducerOf<Self> {
```

Comment:
While some of these `reduce(into:)` needed to migrate to the `body` style to compose with `ifLet`, I've migrated all of them for consistency.